homeassistant/components/google_assistant/trait.py

```diff
@@ -1238,6 +1216,61 @@ async def execute(self, command, data, params, challenge):
             )
 
 
+@register_trait
+class InputSelectorTrait(_Trait):
+    """Trait to set modes.
+
+    https://developers.google.com/assistant/smarthome/traits/inputselector
+    """
+
+    name = TRAIT_INPUTSELECTOR
+    commands = [COMMAND_INPUT]
+
+    SYNONYMS = {}
+
+    @staticmethod
+    def supported(domain, features, device_class):
+        """Test if state is supported."""
+        if domain == media_player.DOMAIN and (
+            features & media_player.SUPPORT_SELECT_SOURCE
+        ):
+            return True
+
+        return False
+
+    def sync_attributes(self):
+        """Return mode attributes for a sync request."""
+        attrs = self.state.attributes
+        inputs = [
+            {"key": source, "names": [{"name_synonym": [source], "lang": "en"}]}
+            for source in attrs.get(media_player.ATTR_INPUT_SOURCE_LIST, [])
+        ]
+
+        payload = {"availableInputs": inputs, "orderedInputs": False}
```

Comment:
Why set orderedInputs to False? The source list is ordered so we should set this to true. This enables to change to the "next" source.

Reply:
I did change this, BUT. I just found references here: https://github.com/actions-on-google/smart-home-nodejs/blob/604fc38a0a0b5eb59860c03df357e5d76345d6d6/src/firestore.ts#L422 on separate commands for next/previous input. They are not documented here: https://developers.google.com/assistant/smarthome/traits/inputselector

I think the documentation for this trait may be invalid.